### DISCUSSION TITLE
xrepo自身有下载缓存吗？

### DISCUSSION THREAD

**Discussion by vkensou** [xmake-io/xmake#5693] [Q&A] [answered]
依赖的包每改一个参数就得重新下载一遍，国内的网络又比较差，下载一次非常耗时。

**Comment by waruqi** [comment]
有，下载完了，就不会再下，顶多就是重新解压

**Reply by vkensou** [reply]
那对于git源有缓存吗？感觉最近在用的时候切下配置要下载好久

**Reply by waruqi** [reply, marked as answer]
切配置走的是不同的包，完全独立的，无法缓存，完全相同的版本，配置才有缓存

**Reply by vkensou** [reply]
如果能缓存下就好了，大多数情况不同配置下载的东西其实是一样的。。。

**Reply by star-hengxing** [reply]
> 那对于git源有缓存吗？感觉最近在用的时候切下配置要下载好久

git url 用上了 partial clone https://github.com/xmake-io/xmake/issues/5507 ，下载应该不会慢
